test/test-applications/integrations/TestApplication.StrongNamed/Program.cs

```diff
@@ -27,5 +27,18 @@ public static void Main(string[] args)
     {
         var command = new Command();
         command.Execute();
+
+        // The "LONG_RUNNING" environment variable is used by tests that access/receive
+        // data that takes time to be produced.
+        var longRunning = Environment.GetEnvironmentVariable("LONG_RUNNING");
+        while (longRunning == "true")
+        {
+            // In this case it is necessary to ensure that the test has a chance to read the
+            // expected data, only by keeping the application alive for some time that can
+            // be ensured. Anyway, tests that set "LONG_RUNNING" env var to true are expected
+            // to kill the process directly.
+            Console.WriteLine("LONG_RUNNING is true, waiting for process to be killed...");
```

Comment:
OutOfMemory + this will never finish 😉

Reply:
Please use: https://github.com/Kielek/opentelemetry-dotnet-instrumentation/blob/4fc8a4a76371e9936863cda273e38f8d55f2407c/test/test-applications/integrations/TestApplication.Smoke/Program.cs#L40-L48

Reply:
On CI, something is actively writing to the standard input so it produces tons of logs.

Reply:
and the test would also have to kill the app...